src/components/DigitalCredentials/CredentialsTermsOfUse.vue

```diff
@@ -0,0 +1,393 @@
+        termination, will survive any termination of these Terms of Use. If any provision of these Terms of Use is
+        invalid, illegal or unenforceable, that provision will be severed from these Terms of Use and all other
+        provisions will remain in full force and effect. These Terms of Use will be governed by and construed in
+        accordance with the laws of the province of British Columbia and the applicable laws of Canada. By accepting
+        these Terms of Use, you consent to the exclusive jurisdiction and venue of the courts of the province of
+        British Columbia, sitting in Victoria, for the hearing of any dispute arising from or related to these Terms
+        of Use and their subject matter.
+      </p>
+    </section>
+    <section>
+      <p class="font-weight-bold">
+        16.&emsp;Definitions
+      </p>
+      <div class="text-justify">
+        <p>
+          In these Terms of Use, the following terms have the following meanings:
+        </p>
+        <p>
+          <span class="font-weight-bold">“BC Services Card”</span> means the Province issued identification card
+          to facilitate access to various government services and programs and which may include information such
+          as an individual's name, photograph, date of birth, and a unique identifier.
+        </p>
+        <p>
+          <span class="font-weight-bold">“Digital Business Card”</span> means the verifiable credential issued to
+          you to identify you as the person who, in relation to the matters set out in section 88(1)(a) to (c) of
+          the <span class="font-italic">Partnership Act</span>, filed a registration statement respecting a sole
+          proprietorship with the Registrar of Companies for the purpose of providing that information to
+          Verifiers.
+        </p>
+        <p>
+          <span class="font-weight-bold">“Province”</span> means His Majesty the King in right of the Province of
+          British Columbia as represented by the Minister of Citizens' Services.
+        </p>
+        <p>
+          <span class="font-weight-bold">“Terms of Use”</span> means the agreement referred to in section 1.
+        </p>
+        <p>
+          <span class="font-weight-bold">“Verifiers”</span> means third parties to whom you present your Digital
+          Business Card for the purpose of allowing those third parties to verify its contents.
+        </p>
+        <p>
+          <span class="font-weight-bold">“Wallet”</span> means software, such as the BC Wallet mobile device app,
+          that you can use to receive, store and present digital credentials such as your Digital Business Card.
+        </p>
+      </div>
+    </section>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+
+@Component({})
+export default class CredentialsTermsOfUse extends Vue {
+  readonly touLink = 'https://www2.gov.bc.ca/gov/content' +
```

Comment:
This link returns a 404. Is that expected?

Reply:
For now yes. It's not published online yet.